.circleci/config.yml

```diff
@@ -10,7 +10,7 @@ executors:
 parameters:
     current_golden_images_hash:
         type: string
-        default: b7380bc16e38864fdd1cbd45b929b801db117a9d
+        default: 78fe764f2f2795d49dc75e83ea32d407f091a972
```

Comment:
I’m not sure your changes would have required this? Can’t look at the VRT diffs right now, but wanted to point out a possible change smell.

Reply:
It fixed pretty much everything once I changed it. I think that because the styles and theme got moved, it was... strugglign with that for some reason? I don't know.

Reply:
I think the number of images were different because of the move.